src/__tests__/collection.spec.ts

```diff
@@ -449,4 +449,17 @@ describe('Collection', () => {
 
     it('should emit the removed document as a change (remote)');
   });
+
+  describe('reload', () => {
+    it('should load documents and emit alldocs', async done => {
```

Comment:
`sync done` is deprecated, and should be avoided

Reply:
But this has been used everywhere in the test. You want me to refactor the tests?